modules/ROOT/pages/salesforce-to-twilio.adoc

```diff
@@ -0,0 +1,454 @@
+. In the *Account SID* field, specify your account SID again. Then, close the card.
+
+== Create data types
+
+The NTO systems all use different formats for data. Because the flow that you are creating needs to pass data through those systems, Flow Designer requires you to give examples of those data types. Because you'll reuse a few of the data types, you can create them at the project level, rather than at the points in the flow where you need them.
+
+=== Create the type for the data sent by Salesforce
+
+. In the Project sidebar, click the plus icon that is next to the heading of the Data Types section.
++
+*Result:* The New Type dialog is opened.
+. Create the data type that represents a Salesforce opportunity and that the Push Topic `ClosedOpps` sends to trigger the flow.
+.. Name the data type ‘Salesforce Opportunity’.
+.. Paste this JSON into the Add your JSON example field.
++
+```
+{
+ "Amount": 15000,
+ "Id": "0061U0000079dWXQAY",
+ "CloseDate": "2019-01-25T00:00:00.000Z",
+ "Name": "Dickenson Mobile Generators"
+}
+```
++
+*Result:* The fields from the example are parsed automatically in the right sidebar of the dialog.
+. Click *Save*.
+
+=== Create the type for the data sent to and returned from the Orders API
+
+Following the same steps, create the following two data types:
+
+==== Name: `Order Information`
+
+The process sends data to the Orders API in this format.
+Here is an example to use in the New Type dialog:
+
+```
+{
+       "OppId": "fsjf3439d993",
+       "OrderAmount": 500000,
+       "orderdate": "2018-11-13T11:15:45",
+       "productInfo": {
+           "productID": "CK111274",
+           "category": "Small",
+           "SKU": "CKS0",
+           "productName": "Old School Classic Kicks",
+           "inventory": 9999
+       }
+}
+```
+
+==== Name: `Order Tracking`
+
+The Orders API returns this information after an order is placed.
+Here is example JSON:
+
+```
+{
+ "OrderID": 50122,
+ "TrackingNo": "Z34342211",
+ "ETA": "2018-11-13T11:15:45"
+}
+```
+
+=== Create the type for the data stored in the products database
+
+Call this data type `Product Information` and use this example JSON:
+
+```
+[
+ {
+   "productID": "CK989603",
+   "category": "Medium",
+   "SKU": "CKS50",
+   "productName": "Air Fly High Quality Sneakers",
+   "inventory": 22333
+ }
+]
+```
+
+== Save the opportunity data into a variable
+
+Because you’ll be using data from the opportunity later in the flow, and cards that follow the trigger will be creating their own outputs, you need to store the opportunity data into a variable.
+
+. Mouse over the arrow between the Salesforce card and the Transform card to change the arrow to a plus sign. Then, click the plus sign to add a new component.
+. Search for and select the ‘Set Variable’ component
+. Name the variable `OppDetails`.
+. Click the *f(x)* button to enable script mode, then set the value to `payload`.
```

Comment:
@jimiljojo @rddheath I've asked the team to enable VFE for the 'Set Var' card. i don't like the idea of teaching users to look for and click on the f(x) button as that is something we are actively working to remove. Instead when they add the 'set var' card they should see the panel of Data Pills on the right and should be able to drag in the 'Payload' pill into the 'value' section of the card.